keyboards/mmd/km75/keyboard.json

```diff
@@ -0,0 +1,262 @@
+{
+    "manufacturer": "MMD",
+    "keyboard_name": "MMD-KM75",
+    "maintainer": "Smartmmd",
+    "bootloader": "stm32duino",
+    "diode_direction": "ROW2COL",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "B3", "pin_b": "A7", "resolution": 2}
+        ]
+    },
+    "features": {
+        "bootmagic": true,
+        "console": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": true,
+        "encoder": true
+    },
+    "matrix_pins": {
+        "cols": ["B0", "C15", "C14", "C13", "A15", "B7", "B6", "B5", "B4", "B12", "B13", "B14", "A14", "B2", "B15"],
+        "rows": ["A0", "A1", "A2", "A3", "A4", "A5"]
+    },
+    "processor": "STM32F103",
+    "rgb_matrix": {
+        "animations": {
+            "solid_color":true,
```

Comment:
This has already been requested to be removed, and has subsequently been added back.
```suggestion
```

Reply:
my fault. Sorry